config/initializers/admin_resource_tabs.rb

```diff
@@ -14,3 +10,17 @@
   config.admin_resource_tabs.register(:theming, name: :users, component: Pageflow::Admin::UsersTab)
   config.admin_resource_tabs.register(:theming, name: :features, component: Pageflow::Admin::FeaturesTab)
 end
+
+Pageflow.after_configure do |config|
+  if config.permissions.show_entry_features_to_non_admin
+    config.admin_resource_tabs.register(:entry,
+                                        name: :features,
+                                        component: Pageflow::Admin::FeaturesTab,
+                                        required_account_role: :manager)
+  else
+    config.admin_resource_tabs.register(:entry,
+                                        name: :features,
+                                        component: Pageflow::Admin::FeaturesTab,
+                                        admin_only: true)
+  end
+end
```

Comment:
We already [have a test](https://github.com/aviav/pageflow/blob/007e4a08566642a6131a6996d2ba0ad326446997/spec/controllers/admin/entries_controller_spec.rb#L45) for original case. Let's also cover the other one.